query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/RequestUtils.java

```diff
@@ -183,7 +185,8 @@ private WebClientBuilder(
               ? new SecureCxfClientFactory<>(url, clientServiceClass)
               : new SecureCxfClientFactory<>(url, clientServiceClass, username, password);
 
-      webClient = clientFactory.getWebClient();
+      webClient =
```

Comment:
❓ Are there any future operations that are executed on the web client that need permissions?

Reply:
I don't believe so. I was not seeing any permission issues bubble up after adding this privileged action.